patches/mocha+8.4.0.patch

```diff
@@ -0,0 +1,4 @@
+diff --git a/node_modules/mocha/bin/mocha b/node_modules/mocha/bin/mocha.js
+similarity index 100%
+rename from node_modules/mocha/bin/mocha
+rename to node_modules/mocha/bin/mocha.js
```

Comment:
The problem here is that if we try to `ts-node-esm node_modules/mocha/bin/mocha` we get an unresolved module error because it doesn't have a .js file ending